skypy/galaxy/luminosity.py

```diff
@@ -9,10 +9,153 @@
 
 
 __all__ = [
+    'absolute_to_apparent_magnitude',
+    'apparent_to_absolute_magnitude',
+    'distance_modulus',
+    'luminosity_in_band',
+    'luminosity_from_absolute_magnitude',
+    'absolute_magnitude_from_luminosity',
     'schechter_lf_magnitude',
 ]
 
 
+def absolute_to_apparent_magnitude(absolute_magnitude, distance_modulus):
+    '''Convert absolute to apparent magnitude.
+
+    Parameters
+    ----------
+    absolute_magnitude : array_like
+        Absolute magnitude M.
+    distance_modulus : array_like
+        Distance modulus DM.
+
+    Returns
+    -------
+    apparent_magnitude : array_like
+        Apparent magnitude M + DM.
+
+    '''
+
+    return np.add(absolute_magnitude, distance_modulus)
+
+
+def apparent_to_absolute_magnitude(apparent_magnitude, distance_modulus):
+    '''Convert apparent to absolute magnitude.
+
+    Parameters
+    ----------
+    apparent_magnitude : array_like
+        Apparent magnitude m.
+    distance_modulus : array_like
+        Distance modulus DM.
+
+    Returns
+    -------
+    absolute_magnitude : array_like
+        Absolute magnitude m - DM.
+
+    '''
+
+    return np.subtract(apparent_magnitude, distance_modulus)
+
+
+@uses_default_cosmology
+def distance_modulus(redshift, cosmology):
+    '''Compute the distance modulus.
+
+    Parameters
+    ----------
+    redshift : array_like
+        Redshift of objects.
+    cosmology : Cosmology, optional
+        The cosmology from which the luminosity distance is taken. If not
+        given, the default cosmology is used.
+
+    Returns
+    -------
+    distmod : array_like
+        The distance modulus m - M for each input redshift.
+    '''
+
+    return cosmology.distmod(redshift).value
+
+
+luminosity_in_band = {
+    'Lsun_U': 6.33,
+    'Lsun_B': 5.31,
+    'Lsun_V': 4.80,
+    'Lsun_R': 4.60,
+    'Lsun_I': 4.51,
+}
+'''Bandpass magnitude of reference luminosities.
+
+These values can be used for conversion in `absolute_magnitude_from_luminosity`
+and `luminosity_from_absolute_magnitude`. The `Lsun_{UBVRI}` values contain the
+absolute magnitude of the sun in Johnson/Cousins bands from [1]_.
```

Comment:
```suggestion
AB absolute magnitude of the sun in Johnson/Cousins bands from [1]_.
```
I assume these are AB magnitudes? In which case state that specifically in the documentation. Otherwise the rest of this PR looks fine.